index.html

```diff
@@ -191,14 +188,55 @@ <h2>Uber Navigation</h2>
                      </ul>
                 </div>
                 <div class="work-button">
-                    <button value="See Project">See Project</button>
+                    <button class="btnx btn3" value="See Project">See Project</button>
                 </div>
             </div>
-            <div class="work-section4">
+            <div class="work-section4 firstdev">
                 <img src="image-resources/Snapshoot Portfolio3.jpg" alt="Snapshoot image">
             </div>
         </section>
     </section>
+    <section class="work-main-container container-fluid p-5">
+        <div class="row bg-light work-remain">
+            <span class="closeit"><i class="fa-solid fa-xmark"></i></span>
+            <div class="col-12">
+                <div class="work-main-heading px-2 py-2">
+                    <h1 class="topic-here">Tonic</h1>
+                    <ul class="list-inline flex">
+                        <li class="list-inline-item">CANOPY</li>
+                        <li class="list-inline-item li-first"><i class="fa-solid fa-circle"></i> Back End Dev</li>
+                        <li class="list-inline-item li-first"><i class="fa-solid fa-circle"></i> 2015</li>
+                    </ul>
+                    <img src="image-resources/SPortfolio.jpg" class="img-fluid w-100 " alt="Logo">
+                </div>
+            </div>
+            <div class="col-12">
+                <div class="row d-flex flex-row">
+                    <div class="col-md-8">
+                        <p class="paragh pb-2">
+                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.
+                        </p>
+                    </div>
+                    <div class=" col-md-4 d-flex flex-column">
+                        <ul class="list-inline ul-work justify-content-center">
+                            <li class="list-inline-item badge bg-info text-primary"><a class= "text-decoration-none text-dark" href="">Html</a></li>
```

Comment:
- [ ] I would kindly suggest you follow the project requirement as much as possible. According to the requirement, it is expected of us to create the pop up card dynamically and remove all the hardcoded code from our HTML file. Kindly remove all the hardcoded code for the main work cards and the pop up cards from your HTML file and be sure you have created them dynamically.

- [ ] Secondly, as suggested by the previous reviewer kindly make the `see live` and `see source` button aligned perfectly to the left with the tech stack in your desktop version. With this you'll be sure you have follow the Figma design as much as possible.

Your Design 👇

![desk-mart](https://user-images.githubusercontent.com/78634841/223698004-94813de9-a6e4-4cb2-bbb6-15dc506eb9af.png)